HISTORY.md

```diff
@@ -10,7 +10,24 @@ As such, a _Feature_ would map to either major (breaking change) or minor. A _bu
 * Features
 * Fixes
 * Misc
+
+### [4.13.0](https://github.com/metricfu/metric_fu/compare/v4.12.0...v4.13.0)
```

Comment:
@andynu The entire line.  Maintainers will add the version when releasing. Until release, it's correct o keep under 'master' which could be renamed to 'unreleased'